scripts/GenerateGlobalHeaderFromSymbols.py

```diff
@@ -55,18 +59,19 @@
         global_var["new_name"] = global_var["new_name"].strip()
         data["globals"].append(global_var)
 
-template_file = "./src/globals.h.j2"
-env = Environment(loader=FileSystemLoader("."))
-template = env.get_template(template_file)
+output_file_h = os.path.join(project_root, "src", "globals.h")
+output_file_c = os.path.join(project_root, "src", "globals.c")
+
+env = Environment(loader=FileSystemLoader(project_root))
+template = env.get_template("/src/globals.h.j2")
 rendered_output = template.render(data)
 
-with open("./src/globals.h", "w", encoding="ascii") as file:
+with open(output_file_h, "w", encoding="ascii") as file:
     file.write(rendered_output)
 
-template_file = "./src/globals.c.j2"
-env = Environment(loader=FileSystemLoader("."))
-template = env.get_template(template_file)
+env = Environment(loader=FileSystemLoader(project_root))
+template = env.get_template("src/globals.c.j2")
```

Comment:
"src" + os.sep + "globals.c.j2" if we want to be consistent. Or use os.path.join

Reply:
Unfortunately https://github.com/pallets/jinja/issues/767 see this issue, I have cleaned it up so if we want to implement a workaround we can